src/getdata.nim

```diff
@@ -217,18 +200,18 @@ proc extractPlace(jsonDataPlace: JsonNode) =
       formatted_address: string
 
   # unmarshall 'jsonData' to object structure 'GeoPlace'
-  let place: GeoPlace = to(jsonDataPlace, GeoPlace)
+  let place = to(jsonDataPlace, GeoPlace)
 
   # check if web site request was handled - if so extract data neded:
   if place.status == "OK":
     # extract one value 'formatted_address' from 'GeoPlace.Results' seq:
-    for item in place[].results:
-      Wthr.placeName = item[].formatted_address
+    for item in place.results:
```

Comment:
Nim has automatic dereference for ``ref`` types so you don't really need to write ``[]``